public/resources/stylesheets/page/_admin.scss

```diff
@@ -44,7 +51,7 @@
 .content-table thead tr {
   background-color: #00ab88;
   color: #000000;
```

Comment:
save the colors in variables, and check if that color are not already in the variables file